### PR TITLE
NYT Article Page Clone

### PR DESCRIPTION
**Pull request by StarSheriff2**
For this project we had to clone the NYT times article page, using HTML and CSS only.

We had to make use of the following concepts:
-  Embedding images
-  Working with forms (buttons in this case)
-  Floats and positioning
-  The box model

We also made wide use of Grid and Flexbox properties.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+      <p>“SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page and drew a double box around it.</p>
+      <p>On Monday, Dr. Guth’s starship came in. Radio astronomers reported that they had seen the beginning of the Big Bang, and that his hypothesis, known undramatically as inflation, looked right.</p>
+      <p>Reaching back across 13.8 billion years to <a class="text-links" href="#">the first sliver of cosmic time</a> with telescopes at the South Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian Center for Astrophysics detected ripples in the fabric of space-time — so-called <a class="text-links" href="#">gravitational waves</a> — the signature of a universe being wrenched violently apart when it was roughly a trillionth of a trillionth of a trillionth of a second old. They are the long-sought smoking-gun evidence of inflation, proof, Dr. Kovac and his colleagues say, that Dr. Guth was correct.</p>
```
Comment (acushlakoncept) [inline index.html:67]: You could wrap all these into `<article>` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+      <p>Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part of the team, said, “This is huge, as big as it gets.”</p>
+      <p>He continued, “This is a signal from the very earliest universe, sending a telegram encoded in gravitational waves.”</p>
+      <p>The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.</p>
+      <p>Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
```
Comment (acushlakoncept) [inline index.html:106]: This also can be wrapped in the `article` tag

Check usage of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) elements.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+          <li><a href="#">New York</a></li>
+          <li><a href="#">Business</a></li>
+          <li><a href="#">Tech</a></li>
+          <li><a href="#">Science</a></li>
+          <li><a href="#">Sports</a></li>
+          <li><a href="#">Obituaries</a></li>
+          <li><a href="#">Today's Paper</a></li>
+          <li><a href="#">Corrections</a></li>
+        </ul>
+      </article>
```
Comment (acushlakoncept) [inline index.html]: This is a wrong semantic usage, use `nav` instead as this part is not an article but navigation links.

Update the same for the rest of the links.

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️
Hello @StarSheriff2 and @brenoxav, Great job! 👏  Both of you have done well on this project.

Impressive Work!
However, there are still some changes that are necessary for this project to be approved.

- [x]  It should have a fixed position (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png))
Screen Shot 2020-10-01 at 6 57 23 PM

I have added some comments below, take a look at it, and fix the requested changes and request another review.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team, this project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:

**Comment(s)**
- Always delete unnecessary comments in your CSS file. For instance, in your reset.ccss file, you have a lot of unnecessary multiline comments.
- Try to use more of semantic HTML5 tags.

Again, congratulations, you can merge to master.
Good luck with your next project.

**I'm @taofeekolalere on slack.
Happy Coding :man_technologist:**

**Pull request merged**